README.md

```diff
@@ -49,6 +49,15 @@ test files can be executed via:
     ruby -Ilib:test test/FILENAME.rb
 
 
+### Usage
+
+These are methods
+
+    client.post(property)
```

Comment:
Once the credentials are stored in the gem, property will be able to call `Rentlinx.client` and reliably get a working client. However, with the way it's currently initialized we have to depend on the user knowing that they have to create a client once with valid attributes before they can do any of the Property methods. Once we've got the gem configuration going we'll be set.